### PR TITLE
Makes User Card Size Equal

### LINKED ISSUE #7402: Ensure the user card height is equal for all
If the user does not have badges and profile picture the user card will shrink.
Screenshot from 2020-01-31 18-42-00

### PR DESCRIPTION
**Pull request by urvashigupta7**
Fixes #7402 

### Before
Screenshot from 2020-02-01 18-08-41

### After
Screenshot from 2020-02-01 18-08-19

### CODE REVIEW

**Comment by urvashigupta7** [comment]
@cesswairimu Why it is failing?

**Comment by urvashigupta7** [comment]
@cesswairimu Done! Please Review

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -102,6 +102,9 @@
     .leaflet-popup-content {
       width: 800px !important;
     }
+    .peoplecard{
```
Comment (Uzay-G) [inline app/views/layouts/application.html.erb]: Could you add a space here. It makes it more readable :smile:

Comment (urvashigupta7) [reply app/views/layouts/application.html.erb]: Sure

**Comment by Uzay-G** [comment]
Could you include a screenshot of your changes? Thanks!

**Comment by urvashigupta7** [comment]
@Uzay-G Done.

**Comment by SidharthBansal** [comment]
Great

**Comment by cesswairimu** [comment]
Awesome :100:

**Pull request merged**